.github/workflows/ci.yaml

```diff
@@ -50,27 +55,23 @@ jobs:
         uses: codecov/codecov-action@v3
 
   build:
-    name: Build charm | ${{ matrix.path }}
-    uses: canonical/data-platform-workflows/.github/workflows/build_charm.yaml@v16.7.0
-    strategy:
-      matrix:
-        path:
-          - .
+    name: Build charm
+    uses: canonical/data-platform-workflows/.github/workflows/build_charm.yaml@v18.0.0
     with:
       cache: true
-      path-to-charm-directory: ${{ matrix.path }}
+
   integration-test:
-    name: Integration test charm | 3.4.4
```

Comment:
Could we keep this? It is helpful to quickly know what juju version we're relying on, so we know when to bump